app/main/posts/views/filters/active-search-filters.html

```diff
@@ -1,3 +1,11 @@
+<div class="form-field saved-search" ng-show="collection">
+    <span class="bug">
+        <svg class="iconic" ng-click="removeCollection(collection, $event)">
```

Comment:
same as above. I chose a more verbose option rather than having a 'removeEntity' to have more control and keep things easy to read , but could be unified..